question_loader.py

```diff
@@ -72,18 +74,21 @@ def __getitem__(self, idx):
         question_image = info['Questions'][0]['images']
         samples1, samples2 = [], []
         for im in question_image:
-            image = Image.open(os.path.join(
-                self.root, dir_name, im['image_url']
-            )).convert('RGB')
-            # Augment the images twice.
-            sample1, sample2 = self.transform(image)
-            samples1.append(sample1)
-            samples2.append(sample2)
+
+            try:
+                image = Image.open(os.path.join(
+                    self.root, dir_name, im['image_url']
```

Comment:
remove these except blocks and change back to the original logic. If there are errors here we need to stop training and debug them, not ignore them